counterpartylib/lib/blocks.py

```diff
@@ -491,59 +561,6 @@ def get_tx_info2(tx_hex, block_parser=None):
     # Decode transaction binary.
     ctx = backend.deserialize(tx_hex)
 
-    def arc4_decrypt(cyphertext):
```

Comment:
This change is significant enough that we need to do a complete `kickstart` to make sure it doesn't change the way any transactions are parsed.